org.sil.scripture-app-builder.yml

```diff
@@ -0,0 +1,114 @@
+# REFERENCES:
+  # Python2.7:  https://github.com/flathub/shared-modules/blob/cc50bdbb34cc3d29bf61cfea5e27cd980b7bda04/python2.7/python-2.7.json
+  # OpenJDK17: https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk17/blob/branch/23.08/org.freedesktop.Sdk.Extension.openjdk17.yaml
+  # Node20:    https://github.com/flathub/org.freedesktop.Sdk.Extension.node20/blob/branch/23.08/org.freedesktop.Sdk.Extension.node20.yaml
+# FIRST-TIME-BUILD: ~30m on 10-20Mb/s download speed
+app-id: org.sil.scripture-app-builder
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+  # - org.freedesktop.Sdk.Extension.openjdk21
+finish-args:
+  # Window x11 socket access
+  - --socket=x11
+  # Audio access
+  - --socket=pulseaudio
+  # Unsure if the program absolutely needs ssh
+  # - --socket=ssh-auth
+  # Or the printer socket
+  # - --socket=cups
+  # Network access
+  - --share=network
+  # XShm access
+  - --share=ipc
+  # All external devices (catch all for devices we interact with and graphical acceleration)
+  - --device=all
+  # Local file browsing access
+  - --filesystem=host
```

Comment:
why is this needed?

Reply:
The application builds mobile apps based on project files that are in the file system. It also needs to browse to add audio files and text files to the project from different locations in the file system.

After a build completes, it uses adb to install the app to an Android device so it needs to connect to the device through USB.

Reply:
The application builds android apps from projects located on the user's computer.  Once the android app is built, it connects via usb to the phone to install and run the app.